test/sendEmail.test.js

```diff
@@ -23,4 +28,28 @@ describe('sendTestEmail function', () => {
     expect(result.envelope.from).toEqual(process.env.EMAIL_USERNAME); // Verify the 'from' address
     expect(result.envelope.to).toEqual([recipientEmail]); // Verify the 'to' address
   });
+
+  it('sends a welcome email with expected content', async () => {
+    let testAccount = await createTestAccount();
+
+    process.env.EMAIL_SERVICE = 'Ethereal';
+    process.env.EMAIL_USERNAME = testAccount.user;
+    process.env.EMAIL_PASSWORD = testAccount.pass;
```

Comment:
I have made some changes, but I messed a bit as it is new to me , could you check please?

Reply:
Akos I am a bit messed up now , according to this      `process.env.EMAIL_SERVICE = 'Ethereal'.` we use Ethereal to test credentials but in env email service I have Outlook. I tried to make jest test to see if we have connection Ethereal successful but  obviously it didn't work. Now I don't understand if we should use Outlook or Ethereal both or one of them?  jest test are not successful for api endpoints line 61 ,97 and 115 , could you please help to understand what I am making in the wrong way ?

Reply:
For tests, we don't want to send the email for real with Outlook. This would cause problems like (1) using up our email quotas, (2) risk of sending an email to a real custom from our testing/automated tests. So instead, we set manually reset the environment variable during our tests only, to use the [Ethereal email service](https://ethereal.email/) which is a fake email service specifically designed to be used for testing.

In development and in production, we use Outlook, and allow our server to use the environment variable set by dotenv, read from the `.env` file.